ui/src/components/Common/InputText.vue

```diff
@@ -0,0 +1,37 @@
+<template>
+    <InputText
+        :id="id"
+        :type="type"
+        :placeholder="placeholder"
+        :class="primeClass === 'sm' ? 'p-inputtext-sm' : (primeClass === 'lg' ? 'p-inputtext-lg' : 'p-inputtext')"
```

Comment:
I'm personally not a fan of nested ternary operators (they tend to be hard to maintain and confusing). Is there a way to express this statement in another way?

Reply:
As discussed, removed the ternary operator and will use direct primevue class